Package.swift

```diff
@@ -42,8 +42,8 @@ let package = Package(
 
     .binaryTarget(
       name: "SwiftFormat",
-      url: "https://github.com/calda/SwiftFormat/releases/download/0.54-beta-5/SwiftFormat.artifactbundle.zip",
-      checksum: "7447986db45a51164d23672c07f971406a4c0589b0c423fcb85e95ed8f8e7e48"),
+      url: "https://github.com/calda/SwiftFormat/releases/download/0.54-beta-6/SwiftFormat.artifactbundle.zip",
+      checksum: "2b7213c898f0914713b5d8a75e4ed7c560552f3a9e69f31dbcd38c99b878d738"),
```

Comment:
@calda how do you find/compute this value? Asking to learn.

Reply:
I usually:
 1. Update the url to point to the new value
 2. Run `rm -rf .build && swift package format`
 3. SPM will print out an error that includes the old checksum and the new checksum
 4. Copy the new checksum into the package manifest

There's also a command to calculate this yourself but I don't know it off the top of my head so use this lazy method instead 😛

Reply:
in case it's helpful: if you have the zip locally, you can always run `xcrun swift package compute-checksum <path_to_zip_file>`